### ISSUE TITLE
FGonzalezLopez / DeepSequencePPI

### ISSUE THREAD

**Issue by Bhawnn22** [FGonzalezLopez/DeepSequencePPI#2]
Please provide the dataset folder also if u can. I'm new to python and bioinformatic so can't find it.
And could u plz explain some steps of code in 2-3 line about what u did in particular section

**Comment by FGonzalezLopez** [comment]
Hi! I'm afraid the datasets are left out on purpose, but they are available at the link provided (click on "Full text" to the right, or follow [this link](https://pubs.acs.org/doi/10.1021/acs.jcim.7b00028) ), there you can download a .zip file with the PPI data. If you want to obtain the AA sequences for each protein as well, they are available at the [download section of Uniprot](https://www.uniprot.org/downloads). The code in the repo uses the .fasta format.

As for the explanation, what section in particular are you referring to?

**Comment by Bhawnn22** [comment]
Hey there, thanks for getting back to me so quickly.
Actually  m from computer science field... I have less knowledge about the
uniprot database.. Nd how and from where to search for AA sequence.
Also the link they provided is in http not https.... I tried to open... But
failed.
I want the dataset just for learning purpose.... If any code runs
successfully then only things get cleared for CS students.
If u dnt want to provide, could u plz provide manipulated data.. That help
me to run this code.

**Comment by FGonzalezLopez** [comment]
> Also the link they provided is in http not https

They work perfectly fine for me, I can't help you any further than giving you the direct links.

1. Cell 8 loads the .csv with the PPI data (which you can find [here](https://pubs.acs.org/doi/suppl/10.1021/acs.jcim.7b00028/suppl_file/ci7b00028_si_001.zip) from the first link as "Supplementary S1.csv")
2. Cell 9 loads the .fasta file containing protein sequence data ([this file](https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/complete/uniprot_sprot.fasta.gz))
3. Cells 10-14 read the data in the .fasta file, and add the sequences to the dataset created in cell 8.

Then you should have a dataset with the protein codes, the interaction label (yes/no) and the raw AA sequences. The rest of code *should* work, unless there has been a change in the libraries (which is more than likely, since the code is almost 3 years old), but I'm afraid you are on your own in that regard. Luckily, the code is native Python, Numpy, Pandas, Sklearn or Keras; out of all of them, you probably won't have to modify much except for Keras.

I'm afraid I haven't kept up with changes in Keras/TF, so this is as much help as I can offer.
